reference/6/Microsoft.PowerShell.Core/Enable-PSRemoting.md

```diff
@@ -21,19 +21,41 @@ Enable-PSRemoting [-Force] [-SkipNetworkProfileCheck] [-WhatIf] [-Confirm] [<Com
 ## DESCRIPTION
 
 The `Enable-PSRemoting` cmdlet configures the computer to receive PowerShell remote commands that
-are sent by using the WS-Management technology.
+are sent by using the WS-Management technology. WS-Management based PowerShell remoting is currently
+supported only on Windows platform.
 
-PowerShell remoting is enabled by default on Windows Server 2012. You can use `Enable-PSRemoting`
-to enable PowerShell remoting on other supported versions of Windows and to re-enable remoting on
-Windows Server 2012 if it becomes disabled.
+PowerShell remoting is enabled by default on Windows Server platforms. You can use
+`Enable-PSRemoting` to enable PowerShell remoting on other supported versions of Windows and to
+re-enable remoting if it becomes disabled.
 
 You have to run this command only one time on each computer that will receive commands. You do not
-have to run it on computers that only send commands. Because the configuration starts listeners, it
-is prudent to run it only where it is needed.
-
-Beginning in PowerShell 3.0, the `Enable-PSRemoting` cmdlet can enable PowerShell remoting
-on client versions of Windows when the computer is on a public network. For more information, see
-the description of the **SkipNetworkProfileCheck** parameter.
+have to run it on computers that only send commands. Because the configuration starts listeners to
+accept remote connections, it is prudent to run it only where it is needed.
+
+Enabling PowerShell remoting on client versions of Windows when the computer is on a public network
+is normally disallowed, but you can skip this restriction by using the **SkipNetworkProfileCheck**
+parameter. For more information, see the description of the **SkipNetworkProfileCheck** parameter.
```

Comment:
You could link this to the parameter section if you wanted. e.g. [SkipNetworkProfileCheck](#-SkipNetworkProfileCheck)

double check that works before committing ;)

Reply:
This change has caused a document validation warning.  So I will remove it.